try {
    //code
} catch ( InterruptedException e) {
    //interrupted
}